de-lib/src/main/java/org/iplantc/de/theme/base/client/fileViewers/callbacks/ShareAnonymousCallbackDefaultAppearance.java

```diff
@@ -9,15 +9,14 @@
 import com.google.gwt.core.client.GWT;
 import com.google.gwt.safehtml.client.SafeHtmlTemplates;
 import com.google.gwt.safehtml.shared.SafeHtml;
-import com.google.gwt.safehtml.shared.SafeUri;
 
 /**
  * @author jstroot
  */
 public class ShareAnonymousCallbackDefaultAppearance implements ShareAnonymousCallback.ShareAnonymousCallbackAppearance {
     interface EnsemblPopupTemplate extends SafeHtmlTemplates {
-        @SafeHtmlTemplates.Template("{0}<img src='{1}' qtip='{2}'></img>")
-        SafeHtml notificationWithContextHelp(SafeHtml label, SafeUri img, String toolTip);
+        @SafeHtmlTemplates.Template("{0}")
```

Comment:
Good point. Although it doesn't hurt anything we can get away with just a text. I will push that change.